examples/tide/hello-world/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tide = "0.16.0"
+sync_wrapper = "0.1"
```

Comment:
I assume this means `tide` also has an internal type that does not implement `Sync`?

Reply:
Yes, @chesedo :thinking:

Reply:
Would you mind confirming this since the [docs](https://docs.rs/tide/latest/tide/struct.Server.html#impl-Sync) say `tide::Server` is `Sync`?